src/Nevermind/Nevermind.Runner/JsonRpcRunner.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel;
+using Nevermind.Core;
+using Nevermind.JsonRpc;
+using Nevermind.JsonRpc.DataModel;
+using Unity;
+using Unity.Wcf;
+
+namespace Nevermind.Runner
+{
+    public class JsonRpcRunner : IJsonRpcRunner
+    {
```

Comment:
let us remove unity, it will slow down startup by a few seconds probably, we can write a simple container, we would not need any advanced scenarios IMHO